File: 프로그래머스/1/12901. 2016년/2016년.py
import datetime

def solution(a, b):
    specific_date = datetime.date(2016, a, b)
    weekday = specific_date.weekday()
    
    weekday_name = ['MON', 'TUE', 'WED', 'THU', 'FRI', 'SAT', 'SUN']
    
    return weekday_name[weekday]

File: 프로그래머스/1/12901. 2016년/2016년.js
function solution(a, b) {
    specificDate = new Date(2016, a - 1, b);
    weekday = specificDate.getDay();
    
    weekdayName = ['SUN', 'MON', 'TUE', 'WED', 'THU', 'FRI', 'SAT'];
    
    return weekdayName[weekday];
}